### PR TITLE
Adds AdobeVideoHeartbeatSDK

### PR DESCRIPTION
**Pull request by ladanazita**
Removes manually added Adobe framework files and links.
	Adds AdobeVideoHeartbeatSDK dependency to podspec.
	Since the dependency is a static lib, updates to
	`--use-libraries` on `pod lib lint`
	Runs formatter.

### CODE REVIEW

**Comment by ladanazita** [comment]
Just for good measure and my own sanity, I imported the ADBMediaHeartbeat header and triggered a sample event, ran the app and saw that it successfully compiled
screenshot 2017-12-06 00 04 54

Also ran `make build`, `make lint` with the above implemented and they succeed 😅

File: Makefile [hunk 1]
```diff
@@ -9,7 +9,7 @@ install: Example/Podfile $(PROJECT).podspec
 	pod install --project-directory=Example
 
 lint:
-	pod lib lint --allow-warnings
+	pod lib lint --allow-warnings --use-libraries
```
Comment (f2prateek) [inline Makefile]: Adobe is still a static library. Shouldn't this flag still be required?

Comment (ladanazita) [reply Makefile]: Hmm not sure what you mean, the flag was added not removed

Comment (f2prateek) [reply Makefile]: Ah ok read the diff wrong.

**Pull request merged**